config/webpack.config.js

```diff
@@ -139,28 +139,18 @@ module.exports = env => {
             {
               loader: MiniCssExtractPlugin.loader,
             },
-            buildOptions.buildtype === ENVIRONMENTS.VAGOVSTAGING
-              ? {
-                  loader: 'css-loader-1.0.0',
-                  options: {
-                    sourceMap: enableCSSSourcemaps,
-                  },
-                }
-              : {
-                  loader: 'css-loader-0.28.10',
-                  options: {
-                    minimize: isOptimizedBuild,
```

Comment:
Minimization was removed from `css-loader` so this follows the [CHANGELOG](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#100-2018-07-06) recommendation to use `cssnano` in `postcss-loader` for css minification.